node/service/src/parachains_db/mod.rs

```diff
@@ -30,15 +30,19 @@ pub(crate) mod columns {
 	}
 
 	pub mod v1 {
-		pub const NUM_COLUMNS: u32 = 5;
+		pub const NUM_COLUMNS: u32 = 6;
```

Comment:
adding a column is a breaking change that requires a migration at least for rocksdb (don't know about parity-db), see
https://github.com/paritytech/polkadot/blob/1c786b372617f26f5982fadc77269e9d4a7c5da2/node/service/src/parachains_db/upgrade.rs#L117-L130

Reply:
good call!

Reply:
actually I refactored this part of the codebase some months ago, but totally forgot to think about this 🙈